File: python/infeccion/recompiler.py
import subprocess

def recompile_apk(input_dir, output_apk):
    result = subprocess.run(['apktool', 'b', input_dir, '-o', output_apk], capture_output=True, text=True)
    if result.returncode == 0:
        print(f"APK recompilado correctamente en {output_apk}")
    else:
        print(f"Error al recompilar el APK: {result.stderr}")

File: java/infeccion/main/java/com/ejemplo/Recompiler.java
package com.ejemplo;

import java.io.IOException;

public class Recompiler {
    public static void recompileApk(String inputDir, String outputApk) {
        try {
            Process process = new ProcessBuilder("apktool", "b", inputDir, "-o", outputApk)
                .inheritIO()
                .start();
            process.waitFor();
            System.out.println("APK recompilado correctamente en " + outputApk);
        } catch (IOException | InterruptedException e) {
            System.out.println("Error al recompilar el APK: " + e.getMessage());
        }
    }
}
